pkg/antctl/raw/proxy/command.go

```diff
@@ -78,11 +79,11 @@ func (o *proxyOptions) validateAndComplete() error {
 	}
 
 	if o.staticDir != "" {
-		fileInfo, err := os.Stat(o.staticDir)
+		fileInfo, err := fsTest.Stat(o.staticDir)
 		if err != nil {
-			klog.Warningf("Failed to stat static file directory %s: %v", o.staticDir, err)
+			klog.InfoS("Failed to stat static file directory", o.staticDir, err)
 		} else if !fileInfo.IsDir() {
-			klog.Warningf("Static file directory %s is not a directory", o.staticDir)
+			klog.InfoS("Static file directory", o.staticDir, "is not a directory")
```

Comment:
these 2 calls to `klog.InfoS` are not correct

The signature for the function is:
```go
func InfoS(msg string, keysAndValues ...interface{})
```

For example, the second one could be:
```go
klog.InfoS("Static file directory is not a directory", name, o.staticDir)
```